Resources/Prototypes/Entities/Objects/Fun/water_balloons.yml

```
@@ -0,0 +1,107 @@
+- type: entity
+  parent: BaseItem
+  id: EmptyWaterBalloon
+  suffix: Empty
+  name: empty water balloon
+  description: Fill it up to reveal it's true color!
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/water_balloons.rsi
+    layers:
+    - state: icon
+  - type: WaterBalloon
+  - type: Item
+    size: Tiny
+  - type: SolutionContainerManager
+    solutions:
+      balloon:
+        maxVol: 10
+  - type: DrawableSolution
+    solution: balloon
+  - type: RefillableSolution
+    solution: balloon
+  - type: DrainableSolution
+    solution: balloon
+  - type: ExaminableSolution
+    solution: balloon
+  - type: Spillable
+    solution: balloon
+
+- type: entity
+  parent: BaseItem
+  id: FilledWaterBalloon
+  suffix: do not spawn :c 
```

why tho?
and if you want it to leave
```suggestion
  suffix: DO NOT SPAWN
```

---

the entity name is kinda a lie bcause its empty, unless it was spawned by filling an empty balloon

so to be safe i clarified that it shouldnt be spawned :33 
doesnt cause any massive errors or anything but its weird